reference/conanfile/attributes.rst

```diff
@@ -817,7 +817,7 @@ This object should be filled in ``package_info()`` method.
 +--------------------------------+---------------------------------------------------------------------------------------------------------+
 | self.cpp_info.cflags           | Ordered list with pure C flags. Defaulted to ``[]`` (empty)                                             |
 +--------------------------------+---------------------------------------------------------------------------------------------------------+
-| self.cpp_info.cppflags         | Ordered list with C++ flags. Defaulted to ``[]`` (empty)                                                |
+| self.cpp_info.cxxflags         | Ordered list with C++ flags. Defaulted to ``[]`` (empty)                                                |
```

Comment:
I wouldn't remove cppflags from here, just document it as deprecated. Rest of the docs are fine without them

Reply:
Yes, you are right, let the people know about the old name. Thanks!